spec/features/version_check_spec.rb

```diff
@@ -1,16 +1,19 @@
-describe "Version check" do
-  let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
-
-  it "occurs if the proposal is modified in after seeing the profile page" do
+describe 'Version check' do
+  it 'occurs if the proposal is modified in after seeing the profile page' do
+    proposal = FactoryGirl.create(:proposal, :with_parallel_approvers)
     login_as(proposal.approvers.first)
-    visit "/proposals/#{proposal.id}"
+    time = Time.current
```

Comment:
this is the same as `TIme.zone,now` - http://api.rubyonrails.org/classes/Time.html#method-c-current

Is your suggestion that I change to `Time.now`? I am SO bad with timezones. Hope to live in a time zone-less world once day

:earth_africa: :watch: